src/concepts/glean/debug_ping_view.md

```diff
@@ -0,0 +1,85 @@
+
+### Supported applications
+
+As for now, the following application ids are supported:
+
+*   `org.mozilla.fenix`
+*   `org.mozilla.reference.browser`
+*   `org.mozilla.samples.glean`
+
+## Where can I see the data?
+
+The data is provided in [this web interface](https://debug-ping-preview.firebaseapp.com/). It lists all recently active devices and updates automatically. You can use your debug identifier to quickly identify your own testing data.
+
+Any data sent from a mobile device usually shows up within 10 seconds, updating the pages automatically.
+
+## Can you give me an example?
+
+For example to send a baseline ping immediately from the reference browser, with a debug identifier of `johndoe-test1`:
+
+```
+adb shell am start -n org.mozilla.reference.browser/mozilla.components.service.glean.debug.GleanDebugActivity \
+  --es sendPing baseline \
+  --es tagPings johndoe-test1
+```
+
+`baseline` pings are also sent automatically by Glean when the application goes to the background. So to check these you can set the tag:
+
+```
+adb shell am start -n org.mozilla.reference.browser/mozilla.components.service.glean.debug.GleanDebugActivity \
+  --es tagPings johndoe-test1
+```
+
+Now whenever you put the application in the background, a `baseline` ping should show up in the web interface.
+
+If you triggered some event recording and want to confirm them you can use the `events`
+
+```
+adb shell am start -n org.mozilla.reference.browser/mozilla.components.service.glean.debug.GleanDebugActivity \
+  --es sendPing events \
+  --es tagPings johndoe-test1
+```
+
+**Note**: Glean will always attempt to collect data for the ping that was requested using the `sendPing` command line switch. However, if no data is recorded by the application, nothing will be sent. The `baseline` ping is _guaranteed_ to always be sent, since it’s populated by Glean itself.
+
+
+## Troubleshooting
+
+If nothing is showing up on the dashboard, it would be useful to check the following:
+
+*   If `adb logcat` reports _”Glean must be enabled before sending pings.”_ right after calling the `GleanDebugActivity`, then the application has disabled Glean. Please check with the application team on how to fix that.
+*   If no error is reported when triggering tagged pings, but the data won't show up on the dashboard, check if the used `<application-id>` is the same expected by the Glean pipeline (i.e. the one used to publish the application on the Play Store).
+*   Before April 12 builds, Fenix had a crashing bug around Glean. This is now fixed, please update.
```

Comment:
Would it make sense to already add a year here?

Reply:
Or we just remove it, it shouldn't affect current users anymore.